website/docs/r/storage_account.html.markdown

```diff
@@ -104,6 +103,9 @@ The following arguments are supported:
 
 * `enable_https_traffic_only` - (Optional) Boolean flag which forces HTTPS if enabled, see [here](https://docs.microsoft.com/en-us/azure/storage/storage-require-secure-transfer/)
     for more information.
+    
+* `is_hns_enabled` - (Optional) Boolean flag which controls if hierarchical namespace is enabled. Used for Azure Data Lake Storage Gen 2, see [here](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account/)
```

Comment:
```suggestion
* `is_hns_enabled` - (Optional) Is Hierarchical Namespace enabled? This can be used with Azure Data Lake Storage Gen 2 ([see here for more information](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account/)). Changing this forces a new resource to be created.
```